src/geo/locate.js

```diff
@@ -19,6 +22,7 @@ const parseResponse = (body) => {
   return {
     type: res.geometry.type,
     coordinates: res.geometry.coordinates,
+    bbox: res.bbox,
```

Comment:
Does this need to be returned/is it useful? This is extra data that will be stored.